xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -1048,7 +1083,7 @@ final class DatabaseService {
         
         return try? database.execute(
             sql: """
-            SELECT id, modified, excerpt
+            SELECT did, id, modified, excerpt
```

Comment:
I would like to add/update a test for readRandomEntryInDateRange to ensure we're getting the data out correctly.

Reply:
Good call, added `Tests_DatabaseService.testReadRandomEntryInDateRange`, turns out this query was broken but it's also not in use by anything in the app currently (it was once used by the "On This Day" geist).